ibis/pandas/udf.py

```diff
@@ -604,3 +526,73 @@ def wrapped(*args):
             return wrapped
 
         return wrapper
+
+
+@pre_execute.register(ops.ElementWiseVectorizedUDF)
+@pre_execute.register(ops.ElementWiseVectorizedUDF, PandasClient)
+def pre_execute_elementwise_udf(
+    op, *clients, scope=None, aggcontet=None, **kwargs
+):
+    """Register execution rules for elementwise UDFs.
+    """
+    input_type = op.input_type
```

Comment:
there are existing tests for this, rigtt? e.g. this is just a namespace refactor

Reply:
Yes, there are existing tests under ibis/pandas/test/test_udf.py